packages/dashboard/src/redux/sagas/trade.js

```diff
@@ -19,6 +19,22 @@ function* getAccountHistory({ payload }) {
   return yield put(tradeActions.getHistoricalQuotesResponse(historicData))
 }
 
+function* getAccountWatchlist({ payload }) {
+  let watchlistData
+  try {
+    watchlistData = yield getWatchlist(payload)
+  } catch (error) {
+    watchlistData = createResponse(
+      error.response?.status,
+      error.message,
+      null,
```

Comment:
Probably going to want this to be `{}` as if this errors you ultimately set the `watchlist` state to this. When you start doing checks on the watchlist data, if its sometimes an object and sometimes null that causes headaches.